src/SCS.jl

```diff
@@ -16,7 +16,12 @@ function __init__()
     if is_unix()
         dlopen(Base.liblapack_name, RTLD_LAZY|RTLD_DEEPBIND|RTLD_GLOBAL)
     end
-    if !(vnum.major == 1 && vnum.minor == 1)
+    if is_linux()
+        if !(vnum.major == 1 && vnum.minor == 2)
```

Comment:
in this way users will end up with newer version of `SCS.jl` without being forced to upgrade the binary `libscsdir.so`